### PR TITLE
Add user profile

### PR DESCRIPTION
**Pull request by chownces**
## Describe your changes
Backend changes for:
- user profile #93
- disable google oauth #86 
- force all users to create Telegram username #67 

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

File: backend/src/database/daos/users/user.dao.service.ts [hunk 1]
```diff
@@ -43,19 +31,7 @@ export class UserDaoService {
...
-    return this.userModel.query(trx).deleteById(userId);
-  }
-
-  async checkUsernameExists(username: string) {
-    return !!(await this.userModel.query().findOne({ username }));
-  }
-
-  async checkTelegramHandleExists(telegramHandle: string) {
-    return !!(await this.userModel.query().findOne({ telegramHandle }));
+    return await this.userModel.query().insertGraph(user);
```
Comment (therizhao) [inline backend/src/database/daos/users/user.dao.service.ts:34]: Curious, what is this insertGraph thing?

File: backend/src/database/seeds/05-Top_Rope_Grades.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries and reset primary key
+  await knex.raw('TRUNCATE TABLE top_rope_grades RESTART IDENTITY CASCADE');
+
+  // Inserts seed entries
+  await knex('topRopeGrades').insert([
+    { name: '4' },
+    { name: '5c' },
+    { name: '5c+' },
```
Comment (therizhao) [inline backend/src/database/seeds/05-Top_Rope_Grades.ts:11]: There's no 5c+. Can remove.

Fyi, for full list of grades see here. https://www.guidedolomiti.com/en/rock-climbing-grades/

Though we don't have to follow it exactly. Current one is good enuf.

File: backend/src/database/seeds/08-Pronouns.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Knex } from 'knex';
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries and reset primary key
+  await knex.raw('TRUNCATE TABLE pronouns RESTART IDENTITY CASCADE');
+
+  // Inserts seed entries
+  await knex('pronouns').insert([
```
Comment (therizhao) [inline backend/src/database/seeds/08-Pronouns.ts:8]: I was expecting this to be single entries. i.e. He, Her, Him, She, etc

Then the climber can select multiple (up to 4), which is how Instagram does it.

Though right now, I thought about it again, it's good enough.

**Review by therizhao — APPROVED** [review]
LGTM. Some nits, you can take note.

File: backend/src/database/daos/users/user.dao.service.ts (hunk 1 above)
Comment (chownces) [reply backend/src/database/daos/users/user.dao.service.ts:34]: abit like mongo insert. lets us specify a nested object, and the ORM will jus create the entries in the related tables for us

**Pull request merged**